azure-template.yml

```
@@ -0,0 +1,41 @@
+# Define defaults
+parameters:
+  python: '3.7'
+  command: test
+  pip_args: .[test]
+  pinned_dependencies: 'numpy'
+  container: ''
+
+jobs:
+
+- job: ${{ format(parameters.name) }}
+  displayName: ${{ format(parameters.display_name) }}
+  pool:
+    ${{ if eq(parameters.os, 'osx') }}:
+      vmImage: macOS 10.13
```

10.13 is no longer supported. 

```suggestion
      vmImage: macOS 10.15
```